components/AddEventForm/index.js

```diff
@@ -31,9 +89,29 @@ export default function AddEventForm({ onSubmit, appointment }) {
     <StyledForm onSubmit={onSubmit}>
       <StyledParagraph>Please fill in the form:</StyledParagraph>
 
-      <label>
-        Type:
-        <select
+      <StyledLabel htmlFor="title"> Title:</StyledLabel>
+      <StyledInput
+        id="title"
+        required
+        maxLength={20}
+        type="text"
+        name="title"
+        defaultValue={appointment ? appointment.title : null}
+      ></StyledInput>
```

Comment:
inputs are self closing:

```suggestion
      <StyledInput
        id="title"
        required
        maxLength={20}
        type="text"
        name="title"
        defaultValue={appointment ? appointment.title : null}
      />
```

Reply:
Done